.pullapprove.yml

```diff
@@ -39,3 +39,17 @@ groups:
       enabled: true
       approve_regex: '^qa-passed'
       reject_regex: '^qa-failed'
+
+  docs-team:
+    conditions:
+      files:
+        - "docs/*"
+        - "*.md"
+        - "*.rst"
+    required: 1
```

Comment:
I do wonder if we need to have atleast 2 users specified here in case of absence. Worst case, we could ask the author of an urgent PR that contains doc changes to split the PR into 2 pieces (and land doc changes later), but `>1` would be best.